src/arguments.h

```diff
@@ -163,6 +163,7 @@ class Arguments {
     long _interval;
     long _alloc;
     long _nativemem;
+    bool _nofree;
```

Comment:
nit: placing `bool` between `longs` in a struct results in 7 bytes waste. Group 1-byte items together; after `_live` will be a good choice.